### DISCUSSION TITLE
Name Squatting

### DISCUSSION THREAD

**Discussion by xySaad** [community/community#128374] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,
I would like to claim the username @xSaad because the account has only one visible repository and it's empty and the account is inactive since 2021.

**Comment by ebndev** [comment, marked as answer]
Hi @x9Saad,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
